protos/bond_transaction.proto

```diff
@@ -0,0 +1,61 @@
+// Copyright 2018 Bitwise IO
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// -----------------------------------------------------------------------------
+
+syntax = "proto3";
+
+import "bond.proto";
+
+message CreateBond {
+
+	// Identifier of the bond (ISIN)
+	string bond_id = 1;
+
+	// Ticker symbol of the organization which issued the bond
+	string issuing_organization_id = 2;
```

Comment:
Would this also be redundant? It could be based on the organization id of the participant who created the transaction.

Reply:
Yeah, we just have to make this a required field for participants, then. In the original implementation, that field was "optional" within the participant object

Reply:
Yeah, it cannot be required because a participant is created before a organization, right? But, the field should be filled  if participants want to submit certain transactions.